### ISSUE TITLE
analytics-c -> libanalytics?

### ISSUE THREAD

**Issue by yields** [segmentio/analytics-c#6]
seems like most c libs get prefixed with `lib` :P
cool initiative btw! :)

**Comment by stevenmiller888** [comment]
True. It comes down to whether we value our own convention (e.g. "analytics-go", "analytics-ruby", etc.) or the convention used in the language the library is written in. I think I slightly value "analytics-c" more for recognition and consistency with our other libraries. But it doesn't matter too much to me :)
